daemon/graphdriver/quota/projectquota.go

```diff
@@ -194,6 +221,24 @@ func setProjectQuota(backingFsBlockDev string, projectID uint32, quota Quota) er
 	return nil
 }
 
+// getQuotaStat - get the quota stat
+// return 2 means quota is on
+func getQuotaStat(backingFsBlockDev string) (int, error) {
```

Comment:
@amir73il  Add `getQuotaStat` is to check whether the system  support ext4 quota. I didn't know much about that bug you mentioned, so if that bug exist,  `getQuotaStat` still return `2`?